fuel-tx/src/transaction/types/script.rs

```diff
@@ -81,11 +81,13 @@ impl Default for Script {
         let script = fuel_asm::op::ret(0x10).to_bytes().to_vec();
 
         Self {
-            gas_price: Default::default(),
-            gas_limit: TxParameters::DEFAULT.max_gas_per_tx,
-            maturity: Default::default(),
+            gas_limit: TxParameters::DEFAULT.max_gas_per_tx >> 1,
```

Comment:
Where does this value come from? Also using `/ 2` instead of a bit shift would be cleaner anyway.